privacyIDEAADFSProvider/AdapterPresentationForm.cs

```diff
@@ -91,7 +99,7 @@ public string GetFormPreRenderHtml(int lcid)
             return null;
         }
 
-        //returns the title string for the web page which presents the HTML form content to the end user
+        /// Returns the title string for the web page which presents the HTML form content to the end user

```

Comment:
Why xml /// comment here? Is it right?

Reply:
because xml is used for documentation. If you put /// at the top of a function, it generates the template (which was to verbose here so i just used a one-liner)